app/domain/user/router.py

```
@@ -1,3 +1,13 @@
+from fastapi import APIRouter, Depends
 
-router = APIRouter()
+from app.api.deps.core import get_current_user
+
+from .model import User
+
+router = APIRouter(prefix="/users")
+
+
+@router.get("/me")
+async def get_me(user: User = Depends(get_current_user)):
```

Ruff (0.15.0)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)